build/charts/antrea/values.yaml

```diff
@@ -323,13 +323,12 @@ whereabouts:
 multicluster:
   # -- Enable Antrea Multi-cluster Gateway to support cross-cluster traffic.
   # This feature is supported only with encap mode.
-  enable: false
+  enableGateway: false
```

Comment:
We did not change enable to enableGateway in 1.10. Please revert all enableGateway changes.

Reply:
Done